### ISSUE TITLE
Travis CI build errored - mozdownload failed with AttributeError: 'module' object has no attribute 'getLogger'

### ISSUE THREAD

**Issue by xabolcs** [mozilla/memchaser#234]
[Latest][1] CI build didn't failed, but errored.

```
$ mozdownload $MOZDOWNLOADPARAMS
Traceback (most recent call last):
  File "/usr/local/bin/mozdownload", line 9, in <module>
    load_entry_point('mozdownload==1.12', 'console_scripts', 'mozdownload')()
  File "/usr/local/lib/python2.7/dist-packages/mozdownload/scraper.py", line 1067, in cli
    build = BUILD_TYPES[options.type](**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mozdownload/scraper.py", line 566, in __init__
    Scraper.__init__(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mozdownload/scraper.py", line 118, in __init__
    self.logger = mozlog.getLogger(' ')
AttributeError: 'module' object has no attribute 'getLogger'
The command "mozdownload $MOZDOWNLOADPARAMS" failed and exited with 1 during .
Your build has been stopped.
```

While I'm was there, Travis CI told me that:
> This job ran on our legacy infrastructure. Please read [our docs on how to upgrade][2]

Before debugging any Python here, how about migrating to the new infrastructure (in a separate Issue)?

Cc: @whimboo 

[1]: https://travis-ci.org/mozilla/memchaser/builds/86506880 "Build #172 has errored."
[2]: http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade "Migrating from legacy to container-based infrastructure"

**Comment by whimboo** [comment]
This is mozdownload and not memchaser, right? I think you want to file this issue on the other repository. :) Btw, the container based infrastructure is easy to do and has a lot of benefits. We already did that for firefox-ui-tests. But yes, this will be a separate issue.

**Comment by xabolcs** [comment]
> This is mozdownload and not memchaser, right?

The Travis build for MemChaser errored.
Of course, something gone wrong with mozdownload.

So at least a mozdownload version bump needed for MemChaser.

**Comment by whimboo** [comment]
Oh! Totally! I see its using version 1.12! Wow... But we have to wait for https://github.com/mozilla/mozdownload/issues/331.

**Comment by xabolcs** [comment]
> [...] But yes, this will be a separate issue.

Issue #235 was filed.

**Comment by xabolcs** [comment]
For the mozdownload update should I stick to `1.18.1` as previously did?

**Comment by whimboo** [comment]
Given that this is for testing with Travis only, I wouldn't mind to be loosely in the version. I would say just depend on mozdownload without any version specified. That would reduce the amount of version bumps in the future.

**Comment by xabolcs** [comment]
Thanks!

Taking care of it.

**Comment by xabolcs** [comment]
Fixed by commit ab100c59e1061d2a717cb13d632794609b274a54!
